test/str_test.py

```diff
@@ -78,7 +78,7 @@ def test_remote_value(self):
             == '<RemoteValue device_name="MyDevice" feature_name="Unknown" GroupAddress("1/2/3")/GroupAddress("1/2/4")/None/None/>'
         )
 
-        remote_value.payload = DPTArray([0x01, 0x02])
+        remote_value.value = DPTArray([0x01, 0x02])
```

Comment:
This is not a valid value for `remote_value.value`. It is for `remote_value.telegram.payload.value`